javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -53,6 +54,7 @@
 
     public static final int STRICTFP = Modifier.STRICT;
 
+    @Deprecated
```

Comment:
mmm, I did not want the new methods to call the old methods because we are going to get rid of the old methods in the future. Doing the opposite would permit to be DRY but the problem is that the old method take an `int` while the new methods take a `NodeWithModifiers`, which I cannot build from an `int`. I mean, I could create an anonymous class to wrap the int and implement NodeWithModifiers but it seems overkilling to me